release/release_tests.yaml

```
@@ -414,6 +414,30 @@
 
   alert: default
 
+- name: air_benchmark_tune_torch_mnist_large_gpu
+  group: AIR tests
+  working_dir: air_tests/air_benchmarks
+
+  frequency: nightly
```

Looks expensive, consider running weekly?
```suggestion
  frequency: weekly
```